internal/test/e2e/cluster.go

```diff
@@ -253,7 +253,7 @@ func newTestCluster(t *testing.T, opts testClusterOptions) *TestCluster {
 	if opts.dir != "" {
 		dir = opts.dir
 	}
-	logger := newTestLogger()
+	logger := zap.NewNop()
```

Comment:
Wanted to draw attention to this change, no logs by default? Mostly because they are very verbose, even on `warn` level which is not great.